src/inventories/batches/InventoryBatchCreationService.ts

```diff
@@ -0,0 +1,60 @@
+import * as Q from 'q';
+import * as uuidV4 from 'uuid/v4';
+import { AxiosRequestConfig } from 'axios';
+
+import { Logger, LogLevel } from 'channelape-logger';
+import LocationsService from '../../locations/service/LocationsService';
+import BatchCreationRequest from '../quantities/model/AdjustmentRequest';
+import Adjustment from '../quantities/model/Adjustment';
+import Resource from '../../model/Resource';
+import Version from '../../model/Version';
+import RequestClientWrapper from '../../RequestClientWrapper';
+import RestService from '../../service/RestService';
+
+export class InventoryBatchCreationService extends RestService {
```

Comment:
I think we can build off of the existing inventory quantities service here. (InventoryBatchAdjustmentsService.ts) Can we just add new methods there for setBatchAsync and adjustBatchAsync? If you don't think that makes sense, I'd rather make this BatchFocused, so we can add any more batch calls here like the GET /v1/batches/:id you added.

Reply:
Also, do you have the tests for this class?